### PR TITLE
MWP-977: Migrated from glide to dep, removed vendor directory from repo, upgrade kube go client from version 2 to 10

### PR DESCRIPTION
**Pull request by russellmacshane**

### CODE REVIEW

**Comment by aalvarado1** [comment]
Gonna do some testing.

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  branch = "master"
+  name = "github.com/vburenin/nsync"
+
+[[constraint]]
+  name = "gopkg.in/gormigrate.v1"
+  version = "~1.2.1"
+
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "10.0.0"
```
Comment (ctriv) [inline Gopkg.toml]: You need to do some extra mojo with this to ensure that the right api-machinery is installed.  https://medium.com/programming-kubernetes/using-gos-dep-to-organize-your-kubernetes-client-go-dependencies-509ddc766ed3 is a good reference.

**Review by ctriv — COMMENTED** [review]
Looks good in general, but this is the sort of change where testing is far more important than code review.

File: Gopkg.toml (hunk 1 above)
Comment (russellmacshane) [reply Gopkg.toml]: Once I get this in, I'll ask Roman to also take a good look at this before merging.

**Comment by aalvarado1** [comment]
As is all tests look good. Tested create, renewals, delete, failed. Also deleted secret and it was recreated upon sync which is good. Create cert with existing deleted cert name also good.

**Review by ctriv — APPROVED** [review]

**Review by aalvarado1 — APPROVED** [review]

**Pull request merged**